commit 2cc88a4539f55960bfa495e9bc388e10f2bdf763
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun May 14 16:37:18 2017 +0200

    Remove gtk+-3 build dep, install versioned gtk.css dirs unconditionally
    
    GTK will search /usr/share/themes/<theme>/gtk-<version> and then earlier
    versions all the way back to 3.0.
    
    See also:
    https://developer.gnome.org/gtk3/stable/GtkCssProvider.html#GtkCssProvider.description

diff --git a/Breeze-dark-gtk/CMakeLists.txt b/Breeze-dark-gtk/CMakeLists.txt
index 15fbb2f..6a82a5e 100644
--- a/Breeze-dark-gtk/CMakeLists.txt
+++ b/Breeze-dark-gtk/CMakeLists.txt
@@ -3,8 +3,6 @@ install(DIRECTORY gtk-2.0 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Bre
 install(DIRECTORY assets DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark
         ${directory_EXCLUDES})
 
-if (PKG_GTK_VERSION VERSION_LESS "3.20")
-install(FILES gtk-3.18/gtk.css DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark/gtk-3.0/)
-else()
-install(FILES gtk-3.20/gtk.css DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark/gtk-3.0/)
-endif()
+install(DIRECTORY gtk-3.0 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark)
+install(DIRECTORY gtk-3.18 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark)
+install(DIRECTORY gtk-3.20 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze-Dark)
diff --git a/Breeze-gtk/CMakeLists.txt b/Breeze-gtk/CMakeLists.txt
index 5096c28..65817d4 100644
--- a/Breeze-gtk/CMakeLists.txt
+++ b/Breeze-gtk/CMakeLists.txt
@@ -3,8 +3,6 @@ install(DIRECTORY gtk-2.0 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Bre
 install(DIRECTORY assets DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze
         ${directory_EXCLUDES})
 
-if (PKG_GTK_VERSION VERSION_LESS "3.20")
-install(FILES gtk-3.18/gtk.css DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze/gtk-3.0/)
-else()
-install(FILES gtk-3.20/gtk.css DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze/gtk-3.0/)
-endif()
+install(DIRECTORY gtk-3.0 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze)
+install(DIRECTORY gtk-3.18 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze)
+install(DIRECTORY gtk-3.20 DESTINATION ${KDE_INSTALL_FULL_DATAROOTDIR}/themes/Breeze)
diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0284ec..1190345 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,9 +10,6 @@ include(FeatureSummary)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR} "${CMAKE_SOURCE_DIR}/cmake")
 find_package(GTKEngine)
 
-find_package(PkgConfig)
-pkg_check_modules(PKG_GTK REQUIRED gtk+-3.0)
-
 include(KDEInstallDirs)
 
 # Exclude these patterns when installing directories as they have no use
